src/Nullable.Extended.Analyzer/Nullable.Extended.Analyzer.Test/Verifiers/CSharpCodeFixVerifier`2.cs

```diff
@@ -41,10 +41,11 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult expe
             => await VerifyCodeFixAsync(source, new[] { expected }, fixedSource, referenceAssemblies);
 
         /// <inheritdoc cref="CodeFixVerifier{TAnalyzer, TCodeFix, TTest, TVerifier}.VerifyCodeFixAsync(string, DiagnosticResult[], string)"/>
-        public static async Task VerifyCodeFixAsync(string source, IEnumerable<DiagnosticResult> expected, string? fixedSource, ReferenceAssemblies? referenceAssemblies = null)
+        public static async Task VerifyCodeFixAsync(string source, IEnumerable<DiagnosticResult> expected, string? fixedSource, ReferenceAssemblies? referenceAssemblies = null, OutputKind? outputKind = null)
         {
             var test = new Test(source, fixedSource, referenceAssemblies);
-
+            if (outputKind is not null)
+                test.TestState.OutputKind = outputKind;
```

Comment:
Since the default of `TestState.OutputKind` is null, there is no need to check `outputKind`